.github/workflows/at_server.yaml

```diff
@@ -685,6 +698,83 @@ jobs:
           name: New Docker image for atsigncompany/virtualenv:vip
           url: ${{ secrets.GOOGLE_CHAT_WEBHOOK }}
           status: ${{ job.status }}
+  
+  # Rollback prod secondary image.
+  rollback_prod_secondary_image:
+    if: ${{ github.event.inputs.rollback_prod_secondary_image == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.2
+
+      # Extract version for docker tag
+      - name: Get version
+        run: echo "VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
```

Comment:
How does this get a correct version for the prod tag we want to roll back to?

Reply:
Fixed